samples/fabric/fabric-cli/src/helpers/helpers.ts

```diff
@@ -228,6 +227,77 @@ const pledgeAsset = async ({
   }
 }
 
+// Used to obtain remote network user certificate from '${networkId}_UsersAndCerts.json' file.
+// This is called during Pledge to get the recipientCert, and during Claim to get the pledgerCert.
+const getUserCertFromFile = (
+  remoteUser: string,
+  remoteNetworkId: string
+) => {
+  const usersAndCertsFile = remoteNetworkId + '_UsersAndCerts.json'
+  const credentialsPath = process.env.MEMBER_CREDENTIAL_FOLDER
+    ? path.resolve(__dirname, process.env.MEMBER_CREDENTIAL_FOLDER, '..')
```

Comment:
I'd suggest to have separate env variable for remoteNetworkUsers now. I'm thinking about what if Member_Credentials_Folder is not `src/data/credentials` but a different folder, then its parent directory can cause issue.

Reply:
Code wise there is no issue as discussed currently. Having a new env variable adds more readability however the downside is the increase in the number of the env variables. So not doing changes here at present, to keep the things simple.

Reply:
Okay can we add this as a TODO? I feel its organized that way. This way it seems one has no control to organize the folders.